fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -3354,6 +3354,9 @@ public class Config extends ConfigBase {
             "materialized views(>10) or query is complex(multi table joins).")
     public static long mv_query_context_cache_max_size = 1000;
 
+    @ConfField(mutable = true)
+    public static boolean enable_materialized_view_concurrent_prepare = true;
```

Comment:
Update the `FE Configuration` docs in english?

Reply:
done